src/arkode/arkode_lsrkstep.c

```diff
@@ -1256,7 +1255,7 @@ int lsrkStep_TakeStepSSPs3(ARKodeMem ark_mem, sunrealtype* dsmPtr, int* nflagPtr
   sunrealtype rs  = (sunrealtype)step_mem->req_stages;
   sunrealtype rn  = SUNRsqrt(rs);
   sunrealtype rat = ONE / (rs - rn);
-  int in          = SUNMIN(SUNRround(rn), INT_MAX);
+  int in          = (int)rn;
```

Comment:
The round shouldn't be removed here since `rn` likely has a fractional part (the other places you removed the round were fine). Otherwise, I don't have further comments and think it's ready to merge.

Reply:
Hmm, on second thought, does `req_stages` have to be a perfect square, in which case this is ok? Either way might be good to double check.

Reply:
`rs` is a perfect square, so the `SUNRsqrt` should return a `double` close enough to make sure cast will produce correct `int` value. I still add `SUNRround` as a safety net.